### ISSUE TITLE
Pedal lines must continue over system breaks

### ISSUE THREAD

**Issue by th-we** [rism-ch/verovio#1582]
Piano pedal lines stop at the end of a system:

![grafik](https://user-images.githubusercontent.com/1147152/88559985-0f724f00-d01d-11ea-8de8-f4bd5d64be0b.png)

They should continue to the same point as without system breaks:

![grafik](https://user-images.githubusercontent.com/1147152/88560112-3cbefd00-d01d-11ea-8153-33f9d68c5172.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<mei xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title/>
   </titleStmt>
   <pubStmt/>
  </fileDesc>
 </meiHead>
 <music>
  <body>
   <mdiv>
    <score>
     <scoreDef meter.count="4" meter.unit="4">
      <staffGrp>
       <staffDef clef.shape="G" clef.line="2" lines="5" n="1"/>
      </staffGrp>
     </scoreDef>
     <section>
      <pb/>
      <sb/>
      <measure n="1">
       <staff n="1">
        <layer n="1">
         <mRest xml:id="start1"/>
        </layer>
       </staff>
       <dir tstamp="1" place="above">only IDs</dir>
       <pedal staff="1" tstamp="1" form="line" dir="down"/>
      </measure>
      <sb/>
      <measure n="2">
       <staff n="1">
        <layer n="1">
         <note pname="g" dur="2" oct="4"/>
         <note pname="g" dur="2" oct="4" xml:id="end2"/>
        </layer>
       </staff>
       <pedal staff="1" dir="up" startid="#end2" form="line"/>
      </measure>
      <sb/>
      <measure n="3">
       <staff n="1">
        <layer n="1">
         <mRest/>
        </layer>
       </staff>
       <dir tstamp="1" place="above">only timestamps</dir>
       <pedal staff="1" tstamp="1" form="line" dir="down"/>
      </measure>
      <sb/>
      <measure n="4">
       <staff n="1">
        <layer>
         <mRest/>
        </layer>
       </staff>
       <pedal staff="1" dir="up" tstamp="3" form="line"/>
      </measure>
      <sb/>
      <measure n="5">
       <staff n="1">
        <layer n="1">
         <mRest/>
        </layer>
       </staff>
       <dir tstamp="1" place="above">start with timestamps, end with IDs</dir>
       <pedal staff="1" tstamp="1" form="line" dir="down"/>
      </measure>
      <sb/>
      <measure n="6">
       <staff n="1">
        <layer>
         <mRest xml:id="end6"/>
        </layer>
       </staff>
       <pedal staff="1" dir="up" startid="#end6" form="line"/>
      </measure>
      <sb/>
      <measure n="7">
       <staff n="1">
        <layer n="1">
         <note pname="g" dur="2" oct="4"/>
         <note pname="g" dur="2" oct="4" xml:id="start7"/>
        </layer>
       </staff>
       <dir tstamp="1" place="above">start with IDs, end with timestamps</dir>
       <pedal staff="1" startid="#start7" form="line" dir="down"/>
      </measure>
      <sb/>
      <measure n="8">
       <staff n="1">
        <layer n="1">
         <mRest/>
        </layer>
       </staff>
       <pedal staff="1" dir="up" tstamp="3" form="line"/>
      </measure>
      <sb/>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

**Comment by rettinghaus** [comment]
@th-we I had the impression that this was working at some point … but maybe not.

**Comment by lpugin** [comment]
I agree with @th-we that `@tstamp2` should be used for encoding pedal with lines. https://github.com/rism-ch/verovio/issues/1424#issue-604527075 This is much reliable and much more MEI like. 
Alternatively, we could use `@next` https://github.com/music-encoding/music-encoding/issues/305#issuecomment-226688405. Willing to support unconnected pedal marks will cause endless problem because of unresolved or illogical cases.

**Comment by rettinghaus** [comment]
By using `@tsamp2` you won't be able to encode pedal bounce: 
![image](https://user-images.githubusercontent.com/7693447/92376289-a9d8ae80-f102-11ea-9ba5-edf75b80a8ee.png)

Also, you wouldn't be able to switch between rendering style with `scoreDef@pedal.style`. 
This is again a question about correct/logical encoding. If you have something like 
`<pedal form="line" dir="up" tstamp="1" tstamp2="m0+5" />`
this doesn't make sense either. If you use `@tstamp2` there should be a schematron rule, that forbids using `@form` and `@dir` at the same time. 
Adding `@next` is ok, but then again: you can't rely on that, if its pointer goes nowhere.

**Comment by lpugin** [comment]
I thought this PR was about the simple cases (down / up). For these I think it should implement the `@tstamp2 (or `@endid) approach you recommend here https://github.com/music-encoding/guidelines/issues/138

We can then treat bounces with the `@next` as suggested here https://github.com/music-encoding/music-encoding/issues/305#issuecomment-226688405. (If next is going nowhere then it means that there is a problem in the encoding and I would not expect a line to be rendered.)

OK?

**Comment by lpugin** [comment]
This issue was fixed with ae3761713fa53c9758eac03a5d2baed8b108514d for pedals encoded with `tstamp2` / `@endid`.
